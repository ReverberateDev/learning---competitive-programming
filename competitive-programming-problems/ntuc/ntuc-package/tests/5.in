1 12206226742910
-1
408
