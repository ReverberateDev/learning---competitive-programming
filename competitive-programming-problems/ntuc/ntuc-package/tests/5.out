4980140511107280
